cumulus/zombienet/zombienet-sdk/tests/zombie_ci/statement_store_bench.rs

```diff
@@ -0,0 +1,726 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Benchmarking statement store performance
+
+use anyhow::anyhow;
+use codec::{Decode, Encode};
+use log::{debug, info, trace};
+use sc_statement_store::{DEFAULT_MAX_TOTAL_SIZE, DEFAULT_MAX_TOTAL_STATEMENTS};
+use sp_core::{blake2_256, sr25519, Bytes, Pair};
+use sp_statement_store::{Channel, Statement, Topic};
+use std::{cell::Cell, collections::HashMap, time::Duration};
+use tokio::time::timeout;
+use zombienet_sdk::{
+	subxt::{backend::rpc::RpcClient, ext::subxt_rpcs::rpc_params},
+	LocalFileSystem, Network, NetworkConfigBuilder,
+};
+
+const GROUP_SIZE: u32 = 6;
+const PARTICIPANT_SIZE: u32 = GROUP_SIZE * 8333; // Target ~50,000 total
+const MESSAGE_SIZE: usize = 512;
+const MESSAGE_COUNT: usize = 1;
+const MAX_RETRIES: u32 = 100;
+const RETRY_DELAY_MS: u64 = 500;
+const PROPAGATION_DELAY_MS: u64 = 2000;
+const TIMEOUT_MS: u64 = 3000;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn statement_store_one_node_bench() -> Result<(), anyhow::Error> {
```

Comment:
Let's add some documentation to each of the tests what scenario is running and what is outputting.